master/buildbot/test/unit/test_stats_service.py

```diff
@@ -389,6 +439,27 @@ def test_capture_data(self):
         )], self.fake_storage_service.stored_data)
 
     @defer.inlineCallbacks
+    def test_capture_data_all_builders(self):
+        self.setup_fake_storage([capture.CaptureDataAllBuilders('test')])
```

Comment:
use same convension => camelCase for methodNames

setup_fake_storage => setupFakeStorage